src/main/resources/Journal.fxml

```diff
@@ -0,0 +1,85 @@
+                  <Label layoutX="14.0" layoutY="14.0" text="First Name:" />
+                  <Label layoutX="171.0" layoutY="14.0" text="Last Name:" />
+                  <TextField layoutX="176.0" layoutY="53.0" prefHeight="57.0" prefWidth="143.0" />
+                  <Label layoutX="331.0" layoutY="14.0" text="Age:" />
+                  <TextField layoutX="331.0" layoutY="53.0" prefHeight="57.0" prefWidth="83.0" />
+                  <Label layoutX="431.0" layoutY="14.0" text="Sex:" />
+                  <ChoiceBox layoutX="431.0" layoutY="54.0" prefWidth="150.0" />
+                  <Label layoutX="14.0" layoutY="114.0" text="Medical Insurance:" />
+                  <TextField layoutX="16.0" layoutY="153.0" prefHeight="57.0" prefWidth="304.0" />
+                  <TextField layoutX="331.0" layoutY="153.0" prefHeight="57.0" prefWidth="83.0" />
+                  <Label layoutX="331.0" layoutY="114.0" text="DOB:" />
+                  <Label layoutX="431.0" layoutY="114.0" text="Primary Doctor:" />
+                  <TextField layoutX="431.0" layoutY="153.0" prefHeight="57.0" prefWidth="172.0" />
+                  <Label layoutX="14.0" layoutY="234.0" text="Mailing Address:" />
+                  <Label layoutX="14.0" layoutY="274.0" text="Street:" />
+                  <TextField layoutX="16.0" layoutY="313.0" prefHeight="57.0" prefWidth="304.0" />
+                  <TextField layoutX="336.0" layoutY="313.0" prefHeight="57.0" prefWidth="304.0" />
+                  <Label layoutX="334.0" layoutY="274.0" text="Town:" />
+                  <ChoiceBox layoutX="651.0" layoutY="314.0" prefWidth="150.0" />
+                  <Label layoutX="651.0" layoutY="274.0" text="State:" />
+                  <TextField layoutX="821.0" layoutY="313.0" prefHeight="57.0" prefWidth="172.0" />
+                  <Label layoutX="821.0" layoutY="274.0" text="Zip Code:" />
+                  <Label layoutX="821.0" layoutY="424.0" text="Zip Code:" />
+                  <TextField layoutX="821.0" layoutY="463.0" prefHeight="57.0" prefWidth="172.0" />
+                  <Label layoutX="651.0" layoutY="424.0" text="State:" />
+                  <ChoiceBox layoutX="651.0" layoutY="464.0" prefWidth="150.0" />
+                  <Label layoutX="334.0" layoutY="424.0" text="Town:" />
+                  <TextField layoutX="336.0" layoutY="463.0" prefHeight="57.0" prefWidth="304.0" />
+                  <TextField layoutX="16.0" layoutY="463.0" prefHeight="57.0" prefWidth="304.0" />
+                  <Label layoutX="14.0" layoutY="424.0" text="Street:" />
+                  <Label layoutX="14.0" layoutY="384.0" text="Billing Address:" />
+                  <Label layoutX="14.0" layoutY="544.0" text="Other Doctors:" />
+                  <ListView layoutX="13.0" layoutY="584.0" prefHeight="145.0" prefWidth="314.0" />
+                  <Button layoutX="347.0" layoutY="583.0" mnemonicParsing="false" text="Add Another Doctor" />
```

Comment:
I pretty much used Hboxes and Vboxes to organize the layout of the textboxes and labels. I don't think this entirely matters, but It still looks nice as is.